keyboards/mechlovin/infinityce/keymaps/default/keymap.c

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2020 mechlovin
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+        LAYOUT_all(
+        KC_ESC,     KC_F1,     KC_F2,     KC_F3,     KC_F4,     KC_F5,     KC_F6,     KC_F7,     KC_F8,     KC_F9,     KC_F10,   KC_F11,      KC_F12,    RGB_MOD,                 KC_PSCR,     KC_SLCK,     KC_PAUS,
+        KC_GRV,     KC_1,      KC_2,      KC_3,      KC_4,      KC_5,      KC_6,      KC_7,      KC_8,      KC_9,      KC_0,     KC_MINS,     RGB_HUD,   KC_BSPC,                 KC_INS,      KC_HOME,     KC_PGUP,
+        KC_TAB,     KC_Q,      KC_W,      RGB_MOD,   KC_R,      KC_T,      RGB_MOD,   KC_U,      KC_I,      KC_O,      KC_P,     KC_LBRC,     KC_RBRC,   KC_SLSH,                 KC_DEL,      KC_END,      KC_PGDN,
```

Comment:
Same with `RGB_MOD` here. Should go to a function layer.

```suggestion
        KC_TAB,     KC_Q,      KC_W,      KC_E,      KC_R,      KC_T,      KC_Y,      KC_U,      KC_I,      KC_O,      KC_P,     KC_LBRC,     KC_RBRC,   KC_SLSH,                 KC_DEL,      KC_END,      KC_PGDN,
```